system-addon/lib/TopSitesFeed.jsm

```diff
@@ -76,7 +76,7 @@ this.TopSitesFeed = class TopSitesFeed {
     const newAction = {type: at.TOP_SITES_UPDATED, data: links};
 
     // Send an update to content so the preloaded tab can get the updated content
-    this.store.dispatch(ac.SendToContent(newAction, action.meta.fromTarget));
+    this.store.dispatch(ac.BroadcastToContent(newAction));
```

Comment:
we could also just take out the getting of the links part in refresh, have a clearHistory function to call that function, and then just get that to broadcast to content?